src/github.com/getlantern/launcher/launcher_windows.go

```diff
@@ -17,24 +18,24 @@ var (
 )
 
 func CreateLaunchFile(autoLaunch bool) {
-	var err error
+	var startupCommand string
+
+	lanternPath, err := osext.Executable()
+	if err != nil {
+		log.Errorf("Could not get Lantern directory path: %q", err)
+		return
+	}
 
 	if autoLaunch {
-		lanternPath, err := osext.Executable()
-		if err != nil {
-			log.Errorf("Could not get Lantern directory path: %q", err)
-			return
-		}
-		err = gowin.WriteStringReg("HKCU", runDir, "Lantern", "\""+lanternPath+"\""+" -startup")
-		if err != nil {
-			log.Errorf("Error inserting Lantern auto-start registry key: %q", err)
-		}
+		// Start Lantern normally.
+		startupCommand = fmt.Sprintf(`"%s" -startup`, lanternPath)
 	} else {
-		// We just set the value to the empty string because the windows registry
-		// library we're using doesn't support RegDeleteValue
-		err = gowin.WriteStringReg("HKCU", runDir, "Lantern", "")
-		if err != nil {
-			log.Errorf("Error removing Lantern auto-start registry key: %q", err)
-		}
+		// Just clear stored proxy settings and quit.
+		startupCommand = fmt.Sprintf(`"%s" -clear-proxy-settings`, lanternPath)
```

Comment:
Good catch indeed @fffw. I guess we can modify the pac.exe to backup and restore original settings instead of destroying them.